addons/l10n_tw_edi_ecpay/tests/test_edi.py

```diff
@@ -0,0 +1,393 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from datetime import datetime
+from unittest.mock import patch
+from urllib.parse import urljoin
+
+from freezegun import freeze_time
+
+from odoo.addons.account.tests.test_account_move_send import TestAccountMoveSendCommon
+from odoo.exceptions import UserError
+from odoo.tests import tagged
+from odoo.tests.common import HttpCase
+
+CALL_API_METHOD = 'odoo.addons.l10n_tw_edi_ecpay.utils.EcPayAPI.call_ecpay_api'
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class L10nTWITestEdi(TestAccountMoveSendCommon, HttpCase):
+
+    @classmethod
+    @TestAccountMoveSendCommon.setup_country('tw')
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.company_data['company'].write({
+            'l10n_tw_edi_ecpay_staging_mode': True,
+            'l10n_tw_edi_ecpay_merchant_id': '2000132',
+            'l10n_tw_edi_ecpay_hashkey': 'ejCk326UnaZWKisg',
+            'l10n_tw_edi_ecpay_hashIV': 'q9jcZX8Ib9LM8wYk',
```

Comment:
You can use random info instead of the official test account data.
As you won't reach the API anyway, it's better to not use real credentials, even if they are public.